TTHAnalysis/python/tools/nanoAOD/susySOS_modules.py

```diff
@@ -301,6 +343,8 @@ def tightLepWZ(lep,year):
 isTightLepVV = lambda : ObjTagger('isTightLepVV', "LepGood", [ lambda lep,year: tightLepVV(lep,year) ]) # relax iso cut
 isTightLepWZ = lambda : ObjTagger('isTightLepWZ', "LepGood", [ lambda lep,year: tightLepWZ(lep,year) ]) # relax iso cut
 
+isFOLep = lambda : ObjTagger('isFOLep', "LepGood", [ lambda lep,year: new_tightLeptonSel_SOS(lep,year) or  new_clean_and_FO_selection_SOS(lep,year) ]) 
```

Comment:
The `isClean*` definitions for muons and electrons should be updated to the new FO definition.

Reply:
The are now obsolete, since we switched to one tight definition everywhere.